components/Dialogs/src/MDCAlertController.m

```diff
@@ -205,10 +179,7 @@ - (BOOL)mdc_adjustsFontForContentSizeCategory {
 - (void)mdc_setAdjustsFontForContentSizeCategory:(BOOL)adjusts {
   _mdc_adjustsFontForContentSizeCategory = adjusts;
 
-  // Update any action buttons to match the alert controller.
-  for (MDCButton *actionButton in self.actionButtons) {
-    actionButton.mdc_adjustsFontForContentSizeCategory = adjusts;
-  }
+  self.alertView.mdc_adjustsFontForContentSizeCategory = adjusts;
 
   if (_mdc_adjustsFontForContentSizeCategory) {
```

Comment:
can we move this to the view and just call its getter? if not, use `if (adjusts)` or alertView's bool.

Reply:
Unfortunately that won't work if alertView hasn't been instantiated yet.